src/clauster/templates/_project_row.html

```
@@ -0,0 +1,191 @@
+                          </span>
+                        </label>
+                      </template>
+                      <!-- cloud option folds into Fire-and-forget -->
+                      <label class="form-check small mt-1 ms-1" x-show="lmode === 'detached'" x-cloak>
+                        <input type="checkbox" class="form-check-input" x-model="lcloud">
+                        <span class="form-check-label">also register on claude.ai</span>
+                      </label>
+                      <!-- optional first prompt for Fire-and-forget -->
+                      <div class="mt-2" x-show="lmode === 'detached'" x-cloak>
+                        <label class="form-label small mb-1">First prompt <span class="text-secondary">(optional)</span></label>
+                        <input class="form-control form-control-sm" x-model="lprompt" placeholder="Initial instruction…">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)